game/src/info/person.rs

```diff
@@ -213,7 +213,10 @@ pub fn trips(
                 .btn_solid_light()
                 .custom_batch(row_btn.clone(), ControlState::Default)
                 .custom_batch(
-                    row_btn.color(RewriteColor::Change(app.cs.inner_panel, app.cs.hovering)),
+                    row_btn.color(RewriteColor::Change(
+                        app.cs.inner_panel,
+                        ctx.style().btn_outline_light.bg_hover,
```

Comment:
Note this uses `btn_outline_light`'s color, even though this otherwise acts like a `btn_solid_light`, but I think it looks better:
![Screenshot from 2021-01-24 12-21-51](https://user-images.githubusercontent.com/1664407/105642467-c02ba080-5e3e-11eb-8e24-d4ea86243e16.png)